providers/ns1/ns1Provider.go

```diff
@@ -173,6 +175,20 @@ func buildRecord(recs models.Records, domain string, id string) *dns.Record {
 			rec.AddAnswer(&dns.Answer{Rdata: strings.Split(fmt.Sprintf("%v %s %s", r.CaaFlag, r.CaaTag, r.GetTargetField()), " ")})
 		} else if r.Type == "SRV" {
 			rec.AddAnswer(&dns.Answer{Rdata: strings.Split(fmt.Sprintf("%d %d %d %v", r.SrvPriority, r.SrvWeight, r.SrvPort, r.GetTargetField()), " ")})
+		} else if r.Type == "NAPTR" {
+			rec.AddAnswer(&dns.Answer{Rdata: []string{
+				fmt.Sprintf("%d", r.NaptrOrder),
```

Comment:
Consider using `strconv.Itoa()` for these.

Reply:
Done, thanks!